.github/workflows/scanner-build.yaml

```diff
@@ -47,6 +84,15 @@ jobs:
         # Prevent fatal error "detected dubious ownership in repository" from recent git.
         git config --global --add safe.directory "$(pwd)"
 
+    - uses: ./.github/actions/handle-tagged-build
+
+    - name: Setup Go build environment for release
+      if: |
+        contains(github.event.pull_request.labels.*.name, 'ci-release-build')
```

Comment:
Could we also have a test for the `ci-release-build` label on this PR?

Reply:
In the "default" with the `ci-release-build` label:

```
CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -trimpath -ldflags="-X github.com/stackrox/rox/scanner/internal/version.Version=4.3.x-809-g9652392bc1" -tags=release -o bin/scanner ./cmd/scanner
```